src/librustc_mir/interpret/place.rs

```diff
@@ -391,12 +391,11 @@ where
             // happens at run-time so that's okay.
             let align = match self.size_and_align_of(base.meta, field_layout)? {
                 Some((_, align)) => align,
-                None if offset == Size::ZERO =>
-                // An extern type at offset 0, we fall back to its static alignment.
-                // FIXME: Once we have made decisions for how to handle size and alignment
-                // of `extern type`, this should be adapted.  It is just a temporary hack
-                // to get some code to work that probably ought to work.
-                {
+                None if offset == Size::ZERO => {
+                    // An extern type at offset 0, we fall back to its static alignment.
```

Comment:
And here the comment actually refers to both the pattern and the arm

Reply:
Kinda... though then the wording would be different: "for extern types at offset 0, we fall back". The current wording first states a fact that was established by the environment ("extern type at offset 0"), and then the conclusion, and this wording makes sense inside the arm, IMO.